technic/machines/switching_station.lua

```diff
@@ -94,8 +99,21 @@ minetest.register_node("technic:switching_station",{
 				-- Network exists and is not overloaded, reactivate network
 				technic.activate_network(network_id)
 				infotext = technic.network_infotext(network_id)
+				meta:set_string("infotext", infotext)
+				-- If mesecon signal enabled and power supply or demand changed then send them via digilines.
+				if mesecons_path and digilines_path and mesecon.is_powered(pos) then
+					local network = technic.networks[network_id]
+					if meta:get_int("supply") ~= network.supply or meta:get_int("demand") ~= network.demand then
+						meta:set_int("supply", network.supply)
+						meta:set_int("demand", network.demand)
+						local channel = meta:get_string("channel")
+						digilines.receptor_send(pos, technic.digilines.rules, channel, {
+							supply = network.supply,
+							demand = network.demand
+						})
+					end
+				end
 			end
-			meta:set_string("infotext", infotext)
```

Comment:
Not sure how this ended up to line 102 but should be moved back or overload messages wont be visible.

Reply:
63ec46e